scripts/commands/pos.lua

```diff
@@ -68,7 +68,7 @@ function onTrigger(player, arg1, arg2, arg3, arg4, arg5)
 
     -- report or move position
     if x == nil or y == nil or z == nil then
-        player:PrintToPlayer(string.format("%s's position: X %.4f  Y %.4f  Z %.4f", targ:getName(), targ:getXPos(), targ:getYPos(), targ:getZPos()))
+        player:PrintToPlayer(string.format("%s's position: X %.4f  Y %.4f  Z %.4f Rot %.4f", targ:getName(), targ:getXPos(), targ:getYPos(), targ:getZPos(), targ:getRotPos()))
```

Comment:
I don't think ths changes are needed, since it's a GM command.

Not oposing to them, mind you, but maybe in a different PR?

Reply:
Worth noting rot ain't a float, its 0-255 as a unsigned integer.

Reply:
ok, I will make another merge request :)